### PR TITLE
Update Symfony packages + recipes+ NPM

### PR DESCRIPTION
**Pull request by melroy89**
Before v1.4.0 release, I would like to update the Symfony package minor releases + recipes.

### CODE REVIEW

File: phpunit.xml.dist [hunk 1]
```diff
@@ -33,7 +33,7 @@
         </exclude>
     </source>
 
-    <extensions>
-        <bootstrap class="DAMA\DoctrineTestBundle\PHPUnit\PHPUnitExtension" />
-    </extensions>
+    <!--<extensions>
```
Comment (melroy89) [inline phpunit.xml.dist:36]: Disable `extensions` element. PHPUnit v10 doesn't have this element.  If this is causing issues with functional tests, then we need to down grade to phpunit v9 again.

Comment (asdfzdfj) [reply phpunit.xml.dist:36]: I just merged #396 and the updated extension block from that patch is already updated for phpunit 10 [according to the test bundle docs](https://github.com/dmaicher/doctrine-test-bundle#using-the-bundle-with-phpunit), so there shouldn't be need to disable this top level `extensions` block

Comment (melroy89) [reply phpunit.xml.dist:36]: Ow let me check again.

**Comment by nobodyatroot** [comment]
For some reason my `package-lock.json` wants to get modified after building, do you see similar behavior? i'm running Symfony CLI version 5.8.11 and  Composer version 2.7.1.

```
diff --git a/package-lock.json b/package-lock.json
index 1d7cf621..671ddeb3 100644
--- a/package-lock.json
+++ b/package-lock.json
@@ -10006,6 +10006,7 @@
             }
         },
         "vendor/symfony/stimulus-bundle/assets": {
+            "name": "@symfony/stimulus-bundle",
             "version": "1.0.0",
             "dev": true,
             "license": "MIT",
@@ -10015,6 +10016,7 @@
             }
         },
         "vendor/symfony/ux-autocomplete/assets": {
+            "name": "@symfony/ux-autocomplete",
             "version": "1.0.0",
             "dev": true,
             "license": "MIT",
@@ -10029,19 +10031,20 @@
             }
         },
         "vendor/symfony/ux-chartjs/assets": {
+            "name": "@symfony/ux-chartjs",
             "version": "1.1.0",
             "dev": true,
             "license": "MIT",
             "devDependencies": {
                 "@hotwired/stimulus": "^3.0.0",
                 "@types/chart.js": "^2.9.34",
-                "chart.js": "^3.4.1 <3.9",
+                "chart.js": "^3.4.1 <3.9 || ^4.0",
                 "resize-observer-polyfill": "^1.5.1",
                 "vitest-canvas-mock": "^0.3.3"
             },
             "peerDependencies": {
                 "@hotwired/stimulus": "^3.0.0",
-                "chart.js": "^3.4.1 <3.9"
+                "chart.js": "^3.4.1 <3.9 || ^4.0"
             }
         }
     }
```

**Comment by melroy89** [comment]
@nobodyatroot It's not due to `composer install`:
```
❯ composer install
Installing dependencies from lock file (including require-dev)
Verifying lock file contents can be installed on current platform.
Nothing to install, update or remove
Generating optimized autoload files
160 packages you are using are looking for funding.
Use the `composer fund` command to find out more!

Run composer recipes at any time to see the status of your Symfony recipes.

Executing script assets:install public [OK]

mbin on  update_deps via  v20.11.1 via 🐘 v8.2.15 took 7ms 
❯ git status
On branch update_deps
Your branch is up to date with 'origin/update_deps'.

nothing to commit, working tree clean
```

However, you are indeed correct. When I run: `npm install && npm run dev`, I do see:

```diff
diff --git a/package-lock.json b/package-lock.json
index 1d7cf621..671ddeb3 100644
--- a/package-lock.json
+++ b/package-lock.json
@@ -10006,6 +10006,7 @@
             }
         },
         "vendor/symfony/stimulus-bundle/assets": {
+            "name": "@symfony/stimulus-bundle",
             "version": "1.0.0",
             "dev": true,
             "license": "MIT",
@@ -10015,6 +10016,7 @@
             }
         },
         "vendor/symfony/ux-autocomplete/assets": {
+            "name": "@symfony/ux-autocomplete",
             "version": "1.0.0",
             "dev": true,
             "license": "MIT",
@@ -10029,19 +10031,20 @@
             }
         },
         "vendor/symfony/ux-chartjs/assets": {
+            "name": "@symfony/ux-chartjs",
             "version": "1.1.0",
             "dev": true,
             "license": "MIT",
             "devDependencies": {
                 "@hotwired/stimulus": "^3.0.0",
                 "@types/chart.js": "^2.9.34",
-                "chart.js": "^3.4.1 <3.9",
+                "chart.js": "^3.4.1 <3.9 || ^4.0",
                 "resize-observer-polyfill": "^1.5.1",
                 "vitest-canvas-mock": "^0.3.3"
             },
             "peerDependencies": {
                 "@hotwired/stimulus": "^3.0.0",
-                "chart.js": "^3.4.1 <3.9"
+                "chart.js": "^3.4.1 <3.9 || ^4.0"
             }
         }
     }
```

**Comment by melroy89** [comment]
NPM diff should be fixed now ;)

**Review by nobodyatroot — APPROVED** [review]

**Review by nobodyatroot — APPROVED** [review]

**Comment by melroy89** [comment]
Not sure what is wrong with doctrine packages again. But as you noticed this is not upgraded, see upstream ticket: https://github.com/doctrine/common/issues/1009

**Pull request merged**